packages/react-chat/src/utils/stylesheet.ts

```diff
@@ -33,7 +33,7 @@ export const useResolveAssistantStyleSheet = (assistant?: AssistantOptions, shad
 
     // inject stylesheet url
     (async () => {
-      await addStyleSheetURL(stylesheet, shadowRoot!).catch((error) => {
+      await addStyleSheetURL(stylesheet, shadowRoot ?? document.head).catch((error) => {
```

Comment:
less strict, we append to the `document.head` if no shadowRoot was passed rather than not rendering at all